CODEOWNERS

```diff
@@ -27,6 +32,9 @@
 
 # ==== Online Stores ====
 
+# HBase
+/sdk/python/feast/infra/online_stores/hbase.py @sudohainguyen
+
```

Comment:
`sdk/python/feast/infra/online_stores/contrib/hbase_online_store`
I believe there should be this one as well